themes/hyde/layouts/partials/sidebar.html

```diff
@@ -11,7 +11,7 @@
       {{ range .Site.Menus.main -}}
         <li><a href="{{.URL}}"> {{ .Name }} </a></li>
       {{- end }}
-      <li><a class="fork"> Fork on GitHub</a></li>
+      <li><a href="https://github.com/census-instrumentation/opencensus-website" class="fork"> Fork on GitHub</a></li>
```

Comment:
I think we should give a link to https://github.com/census-instrumentation from there.

We can have an additional "Edit this page on GitHub" at the bottom every page and give link to the website repo.

Reply:
Sounds good. I'll try to fix this in a way that looks nice